libid/CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ add_library(libid
     3d.cpp include/3d.h
     line3d.cpp include/line3d.h
     plot3d.cpp include/plot3d.h
+    pert_engine.cpp include/pert_engine.h
+    pert_function.cpp
+    perturbation.cpp
+    include/point.h
+    Complex.cpp include/Complex.h
```

Comment:
I agree that we need to consolidate complex. Let's just get what we have working for ID 1.1 and then consolidate all the code as it may take some effort, especially for the c-style code.

How does one edit a commit message?

Reply:
Just add new commits to your branch and push them to github and the changes will be reflected on this pull request.

I'm not talking about changing all the usages of complex numbers in the code; I'm talking about this new code.  So you should:

- Delete libid/include/Complex.h
- Delete libid/Complex.cpp
- Delete mention of these files from `CMakeLists.txt`
- Include `<complex>` instead of `<Complex.h>` in your new files
- Use `std::complex<float>` or `std::complex<double>` as appropriate in your new code

This should get the clear up most if not all of the build failures this pull request is generating right now.  We need to have the build clean before it can be integrated into the master branch.

Reply:
I have removed Complex.cpp and Complex.h
Replace Complex with std::complex<double>

Perturbation and Tierazon fractals now use standard C++ complex functions. 
Perturbation runs on standard plotting modes as well as perturbation method. Fractals include a small number of Mandelbrot derivatives.
Art Matrix cubic fractal has been added as well as selected Tierazon fractals.